packages/golden_toolkit/test/multi_screen_golden_test.dart

```diff
@@ -7,20 +7,68 @@
 /// ***************************************************
 
 import 'dart:io';
+import 'package:flutter/material.dart';
 import 'package:golden_toolkit/golden_toolkit.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import 'helper_widgets.dart';
 import 'sample_weather_widget.dart';
 
+const someDuration = Duration(milliseconds: 100);
+
 Future<void> main() async {
   group('Multi Screen Golden examples', () {
     testGoldens('Example of testing a responsive layout', (tester) async {
-      await tester.pumpWidgetBuilder(WeatherForecast());
+      await tester.pumpWidgetBuilder(const WeatherForecast());
       await multiScreenGolden(
         tester,
         'weather_forecast',
         skip: !Platform.isMacOS,
       );
     });
+    group('Responseive layout when image changes depending on layout', () {
+      testGoldens('default configuration', (tester) async {
```

Comment:
I think we need to add some explanation here... this test is to really show that images WON'T be properly loaded if we use the default settings. We should make this more clear.

how about "Demonstrate that images won't be loaded for device variations by default"?

Reply:
actually that's a bad name

Reply:
"Images should appear in Goldens even if not available on first run loop"?
"Should render images in multiScreenGoldens that require additional setup"?